x-pack/legacy/plugins/canvas/public/components/expression_input/reference.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { CanvasFunction, CanvasArgValue } from '../../../types';
+
+export function getFunctionReferenceStr(fnDef: CanvasFunction) {
```

Comment:
Same.  Seems magical, and that there's a doc online somewhere that would explain what this is and how it applies.